### PR TITLE
Add labs folder from hackclub repo

### PR DESCRIPTION
**Pull request by MaxWofford**
This adds the files from https://github.com/hackclub/hackclub/pull/541

### CODE REVIEW

**Comment by MaxWofford** [comment]
@zrl: would you please review this?

**Comment by MaxWofford** [comment]
It goes hand in hand with hackclub/hackclub#541

File: labs/rejection-email.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+That said, we would like to invite your club to hackEDU. We love the community
+you're building at your school and want to invite you to join the hackEDU
+Leaders community: [https://www.facebook.com/groups/1501083703514499]
+
+Introduce yourself by sharing one challenge you know you'll face with your club
+this year.
+
+Looking forward to watching the coding community grow at your school,
+
+Dave
```
Comment (zachlatta) [inline labs/rejection-email.md]: Can you name this file `rejection_email.md`?

**Comment by zachlatta** [comment]
Can you also add the `README.md` and `SETUP.md` files from `hackclub/hackclub:meta/labs/` to this PR?

**Comment by MaxWofford** [comment]
@zachlatta Addressed comments!

**Pull request merged**